data/styles/default/include/priorities_4_overlays.prio.txt

```diff
@@ -368,6 +368,9 @@ amenity-charging_station-motorcar                   # icon z14- (also has captio
 amenity-fuel                                        # icon z14- (also has caption(optional) z14-)
 === 4270
 
+highway-traffic_signals                             # icon z19-
+=== 4260
```

Comment:
Too high a priority for the main style? I think it should be very low like 120 which is just under streetside parkings.
WDYT?

Reply:
Fixed